pkg/views/workspace/create/workspacedata.go

```diff
@@ -0,0 +1,157 @@
+// Copyright 2024 Daytona Platforms Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package create
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/huh"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/daytonaio/daytona/internal/util"
+	"github.com/daytonaio/daytona/pkg/views"
+)
+
+var doneCheck bool
+
+type WorkspaceDataModel struct {
+	lg                      *lipgloss.Renderer
+	styles                  *Styles
+	form                    *huh.Form
+	basicViewActive         bool
+	width                   int
+	quitting                bool
+	showConfigurationOption bool
+}
+
+func GetWorkspaceDataFromPrompt(suggestedName string, workspaceNames []string, showConfigurationOption bool) (string, string, string) {
+	workspaceName, containerImageUrl, osUser := suggestedName, "", ""
```

Comment:
The initial values for `containerImage` and `containerUser` should come from the server default values.

Additionally, these values should be overridable via CLI flags. Currently, there wouldn't be a way for a user to override these values in a non-interactive environment.